pkg/config/template/renderer_test.go

```diff
@@ -72,20 +64,16 @@ func TestParseTemplate(t *testing.T) {
 				"id",
 				invalidTemplateString,
 			},
-			want:    nil,
 			wantErr: true,
 		},
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			got, err := ParseTemplate(tt.args.id, tt.args.content)
+			_, err := ParseTemplate(tt.args.id, tt.args.content)
 			if (err != nil) != tt.wantErr {
 				t.Errorf("ParseTemplate() error = %v, wantErr %v", err, tt.wantErr)
 				return
 			}
-			if !reflect.DeepEqual(got, tt.want) {
```

Comment:
why did you remove asserting the actual template here? Also shouldn't this test move to template_test.go along with the function move?